Source/JRecord_Utilities/JRecord_SchemaCompare/pom.xml

```diff
@@ -15,10 +15,10 @@
             <groupId>net.sf.jrecord.jrecord-parent.jrecord-project-parent</groupId>
             <artifactId>jrecord-base</artifactId>
         </dependency>
-        <dependency>
-            <groupId>net.sf.jrecord.jrecord-parent.jrecord-utilities-parent</groupId>
-            <artifactId>jrecord-cbl2json</artifactId>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
this is commented out in other places and doesn't seem to be needed, I can remove if required

Reply:
yes, it should not be there. I moved it to a seperate project